plugins/storage/object/cloudian/src/main/java/org/apache/cloudstack/storage/datastore/util/CloudianHyperStoreUtil.java

```diff
@@ -0,0 +1,218 @@
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagement;
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagementClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.cloud.utils.exception.CloudRuntimeException;
+
+public class CloudianHyperStoreUtil {
+
+    /** The name of our Object Store Provider */
+    public static final String OBJECT_STORE_PROVIDER_NAME = "Cloudian HyperStore";
+
+    public static final String STORE_KEY_PROVIDER_NAME = "providerName";
+    public static final String STORE_KEY_URL           = "url";
+    public static final String STORE_KEY_NAME          = "name";
+    public static final String STORE_KEY_DETAILS       = "details";
+
+    // Store Details Map key names - managed outside of plugin
+    public static final String STORE_DETAILS_KEY_USER_NAME    = "accesskey";   // admin user name
+    public static final String STORE_DETAILS_KEY_PASSWORD     = "secretkey";   // admin password
+    public static final String STORE_DETAILS_KEY_VALIDATE_SSL = "validateSSL";
+    public static final String STORE_DETAILS_KEY_S3_URL       = "s3Url";
+    public static final String STORE_DETAILS_KEY_IAM_URL      = "iamUrl";
+
+    // Account Detail Map key names
+    public static final String KEY_ROOT_ACCESS_KEY    = "hs_AccessKey";
+    public static final String KEY_ROOT_SECRET_KEY    = "hs_SecretKey";
+    public static final String KEY_IAM_ACCESS_KEY     = "hs_IAMAccessKey";
+    public static final String KEY_IAM_SECRET_KEY     = "hs_IAMSecretKey";
+
+    public static final int DEFAULT_ADMIN_PORT = 19443;
+    public static final int DEFAULT_ADMIN_TIMEOUT_SECONDS = 10;
+
+    public static final String IAM_USER_USERNAME = "CloudStack";
+    public static final String IAM_USER_POLICY_NAME = "CloudStackPolicy";
+    public static final String IAM_USER_POLICY;
+    static {
+        StringBuilder sb = new StringBuilder();
+        sb.append("{\n");
+        sb.append("  \"Version\": \"2012-10-17\",\n");
+        sb.append("  \"Statement\": [\n");
+        sb.append("    {\n");
+        sb.append("      \"Sid\": \"AllowFullS3Access\",\n");
+        sb.append("      \"Effect\": \"Allow\",\n");
+        sb.append("      \"Action\": [\n");
+        sb.append("        \"s3:*\"\n");
+        sb.append("      ],\n");
+        sb.append("      \"Resource\": \"*\"\n");
+        sb.append("    },\n");
+        sb.append("    {\n");
+        sb.append("      \"Sid\": \"ExceptBucketCreationOrDeletion\",\n");
+        sb.append("      \"Effect\": \"Deny\",\n");
+        sb.append("      \"Action\": [\n");
+        sb.append("        \"s3:createBucket\",\n");
+        sb.append("        \"s3:deleteBucket\"\n");
+        sb.append("      ],\n");
+        sb.append("      \"Resource\": \"*\"\n");
+        sb.append("    }\n");
+        sb.append("  ]\n");
+        sb.append("}\n");
+        IAM_USER_POLICY = sb.toString();
+    }
```

Comment:
Why not simply define the variable with the plain string value? Is there any reason to use StringBuilder here?

Reply:
There is no strong reason except that I was frustrated there was no """ in Java11 :-) I can rework it into a standard string and remove the static initialization I guess.